langlib/lang.string/src/main/java/org/ballerinalang/langlib/string/EqualsIgnoreCaseAscii.java

```diff
@@ -59,13 +56,14 @@ public static boolean equalsIgnoreCaseAscii(BString s1, BString s2) {
     }
 
     private static boolean isPureAscii(String  str) {
```

Comment:
I think we can replace this method's entire body with the following line
`return StandardCharsets.US_ASCII.newEncoder().canEncode(str);` Shall we please check this?